project/password/templates/registration/password_change.html

```diff
@@ -0,0 +1,52 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<style type="text/css">
+  .form-signin {
+    width: 100%;
+    max-width: 330px;
+    padding: 15px;
+    margin: auto;
+  }
+  .form-signin .checkbox {
+    font-weight: 400;
+  }
+  .form-signin .form-control {
+    position: relative;
+    box-sizing: border-box;
+    height: auto;
+    padding: 10px;
+    font-size: 16px;
+  }
+  .form-signin .form-control:focus {
+    z-index: 2;
+  }
+  .form-signin input[type="password"] {
+    margin-bottom: 10px;
+    border-bottom-right-radius: 0;
+    border-bottom-left-radius: 0;
+  }
+  .h3{
+    text-align: center;
+  }
```

Comment:
We probably don't need any custom styles at this point. Let's just stick to using Bootstrap classes for [alignment](https://getbootstrap.com/docs/5.0/utilities/text/#text-alignment), [sizing](https://getbootstrap.com/docs/5.0/utilities/text/#font-size), [margin/padding](https://getbootstrap.com/docs/5.0/layout/utilities/#margin-and-padding), etc.
```suggestion
```